lib/pbench/cli/server/pbench_create_user.py

```diff
@@ -0,0 +1,94 @@
+import click
```

Comment:
Again, I think this file should be `pbench-user-create`, at least. We don't want to start using underscores, and we should be moving towards the verb-noun structure to make click subcommand structuring look cleaner.

However, I also note that all of Chuck's commands just have basic object names instead of full command names, which are provided by the `setup.cfg` file `console_scripts` variable as in `pbench-server = pbench.cli.server.shell:main`.

My only concern is that I'm not entirely sure how to employ this sort of command in the cli unit tests since it's going to require a setup build.

Reply:
As you observed, @dbutenhof, the _implementation_ of the command is separated from the details of how it is invoked (which, to a large extent, are specified in the `setup`).  This means among other things, that the module name can include underscores where the command verb uses hyphens (and, I think that this is actually conventional).

Nevertheless, I concur that we should make an effort to move toward noun-verb structure, although, again, that doesn't _necessarily_ have to be reflected in the implementation (but, it would probably be better if it were).

As for unit tests, if a setup build is inconvenient, you can probably invoke the command's implementation directly, or you can run it the same way that Nikhil's unit tests for the command itself invoke it, via a [Click "runner"](https://click.palletsprojects.com/en/7.x/api/#testing).

Reply:
Another thing, having hyphens instead of underscore is not a valid python identifier, I can not seem to import it into another modules like tests. It is also mentioned in the PEP8 guide that we should be using underscore if at all necessary https://www.python.org/dev/peps/pep-0008/#package-and-module-names

Reply:
Yes, if you're importing from the module, you're probably right. But we don't use the `pbench` prefix on any other internal modules, and it's redundant anyway given the `pbench.cli.server` path. So how about just `user_create.py` and map the CLI `pbench-user-create` through `setup.cfg` as the others are done.